pandas-stubs/core/series.pyi

```diff
@@ -1094,8 +1094,14 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     def __div__(self, other: Union[num, _ListLike, Series[S1]]) -> Series[S1]: ...
     def __eq__(self, other: object) -> Series[_bool]: ...  # type: ignore
     def __floordiv__(self, other: Union[num, _ListLike, Series[S1]]) -> Series[int]: ...
+    @overload
     def __ge__(self, other: Union[num, _ListLike, Series[S1]]) -> Series[_bool]: ...
+    @overload
+    def __ge__(self, other: Timestamp) -> Series[_bool]: ...
```

Comment:
I think this may be easier to do without using overloads.  E.g., for `__ge__`, use:
```python
def __ge__(self, other: Union[num, Timestamp, _ListLike, Series[S1]]) -> Series[_bool]: ...
```

Reply:
Right yes, thanks for the suggestions! Ill have a look tomorrow!

Reply:
I added the comparisons the way you suggested, as well as the testing with the use of `assert_type()`